docs/source/using/profiles.adoc

```diff
@@ -8,18 +8,45 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state information.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+Profiles share nothing between them. Things like cache, add-ons and persistent config are specific to a single profile.
```

Comment:
One sentence per line. In this case we might want to drop "Profiles share nothing between them". This is implied by "completely independent"

Reply:
"Things like cache, add-ons and persistent config are specific to a single profile." -> "In particular cached data, add-ons and persistent configuration are scoped per profile."